app/controllers/hyrax/collections_controller.rb

```diff
@@ -2,6 +2,8 @@ module Hyrax
   class CollectionsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
     include Hyrax::BreadcrumbsForCollections
+    layout :decide_layout
+    load_and_authorize_resource except: [:index, :show, :create], instance_name: :collection
```

Comment:
@mcritchlow: Good question! I think action `:create` is expected to have its's own authorization before creating a new collection,  see [collections_controller_behavior.rb#L103](https://github.com/samvera/hyrax/blob/master/app/controllers/concerns/hyrax/collections_controller_behavior.rb#L103).